### PR TITLE
ism330dhcx driver: Fixed most mcuastyle errors

### PR DESCRIPTION
**Pull request by cparata**
Fixed most mcuastyle errors in the source and header files.

Signed-off-by: Carlo Parata <carlo.parata@st.com>

### CODE REVIEW

File: ism330dhcx_reg.h [hunk 1]
```diff
@@ -2836,12 +2836,12 @@ typedef union
...
-                                   uint16_t len);
-int32_t __weak ism330dhcx_write_reg(stmdev_ctx_t *ctx, uint8_t reg,
-                                    uint8_t *data,
-                                    uint16_t len);
+int32_t ism330dhcx_read_reg(stmdev_ctx_t *ctx, uint8_t reg,
+                            uint8_t *data,
+                            uint16_t len);
+int32_t ism330dhcx_write_reg(stmdev_ctx_t *ctx, uint8_t reg,
+                             uint8_t *data,
+                             uint16_t len);
```
Comment (avisconti) [inline ism330dhcx_reg.h:2845]: Ciao Carlo,

Can you possibly put this change in a separate commit? I'd prefer to have this change separated from the rest of warnings. Thanks! (rest is absolutely fine!)

**Review by avisconti — CHANGES_REQUESTED** [review]

File: ism330dhcx_reg.h (hunk 1 above)
Comment (cparata) [reply ism330dhcx_reg.h:2845]: Ok, now it should be fine

**Review by avisconti — APPROVED** [review]
Perfect, thanks!
